css/decanter.css

```diff
@@ -1989,4 +1910,465 @@ a.decanter-button--unstyled {
   border-bottom: 1px solid #b6b1a9;
   border-left: 1px solid #b6b1a9;
   border-right: 1px solid #b6b1a9; }
+
+.page-layout {
```

Comment:
This leads to a good question. How much of this page layout code do we want in the default compiled css? Would we rather have it optional?

Reply:
If the code makes up the page layouts that we've agreed on, I would lean towards it living in the default compiled CSS. Of course, we'll want to be sure that the layouts can be overridden like other things in partials. Were you thinking we might not want to do this? Happy to hear the other side of the argument.